### PR TITLE
Fix home

### PR DESCRIPTION
**Pull request by fortanza**
Layout CSS Mofification

### CODE REVIEW

**Review by John-athan14 — DISMISSED** [review]

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Nagivation from './Navigation/Navigation';
 
 const Header = () => {
   return (
-    <header className="header">
+    <header className={['header', 'container'].join(' ')}>
       <Logo />
       <Nagivation />
```
Comment (Hacene010) [inline src/Components/Header/Header.jsx]: Nagivation or Navigation ??

File: src/Components/SectionCarousel/SectionCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import './SectionCarousel.css';
+
+const SectionCarousel = () => {
+  return (
+    <section className="Slide-Caroussel">
```
Comment (Hacene010) [inline src/Components/SectionCarousel/SectionCarousel.jsx]: plz use the camelCase like the other classNames !

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some changes to do before merging. Plz fix them

File: src/Components/Header/Header.jsx (hunk 1 above)
Comment (fortanza) [reply src/Components/Header/Header.jsx]: The problem is resolved.

File: src/Components/SectionCarousel/SectionCarousel.jsx (hunk 2 above)
Comment (fortanza) [reply src/Components/SectionCarousel/SectionCarousel.jsx]: The problem is resolved.

**Review by John-athan14 — APPROVED** [review]

File: src/Components/SectionCarousel/SectionCarousel.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.SlideCaroussel {
```
Comment (Hacene010) [inline src/Components/SectionCarousel/SectionCarousel.css:1]: Use camelCase everywhere plz

File: src/Components/SectionCarousel/SectionCarousel.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import './SectionCarousel.css';
+
+const SectionCarousel = () => {
+  return (
+    <section className="SlideCaroussel">
```
Comment (Hacene010) [inline src/Components/SectionCarousel/SectionCarousel.jsx:5]: same here

**Review by Hacene010 — APPROVED** [review]
It's ok but think about the camelCase to be coherent plz (see comments)

**Pull request merged**